extensions/amp-ad-exit/0.1/config.js

```diff
@@ -37,7 +38,16 @@ export let AmpAdExitConfig;
 export let NavigationTargetConfig;
 
 /**
- * @typedef {!Object<string, {defaultValue: (string|number|boolean)}>}
+ * @typedef {{
+ *   defaultValue: (string|number|boolean),
+ *   vendorAnalyticsSource: (string|undefined),
+ *   vendorAnalyticsResponseKey: (string|undefined)
+ * }}
+ */
+export let Variable;
```

Comment:
who's importing it? Use `VariableDef` and `VariablesDef`. 
Also is Variable sounds like a too general name to me. Can we rename it?

Reply:
Changed to ...Def here, and in the other PR (#11462). This is used more there, but a little difficult to extricate since both PRs change amp-ad-exit.js and config.js

This is @clawr 's code so adding him for opinions on renaming.